task-cycles.js

```diff
@@ -20,9 +24,13 @@ const arrayOfMultiples = (num, length) => {
 
 // Your code:
 const changeDirection = (array) => {
-// ... write code ...
+    let reverse = []
+    for (let i = array.length - 1; i >= 0; i--){
+        reverse.push(array[i])
+    }
+    return reverse
```

Comment:
We could also use just https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse